source/config/profileUpgradeSteps.py

```diff
@@ -493,9 +493,10 @@ def _convertTypingEcho(profile: ConfigObj, key: str) -> None:
 		log.debug(f"'{key}' not present in config, no action taken.")
 		return
 	except ValueError:
-		log.error(f"'{key}' is not a boolean, no action taken.")
+		log.error(f"'{key}' is not a boolean, deleting.")
```

Comment:
Maybe it's worth mentioning the value?

```suggestion
		log.error(f"'{key}' is not a boolean ({profile["keyboard"][key]}), deleting.")
```